esphomeyaml/cookbook/h801.rst

```diff
@@ -0,0 +1,96 @@
+H801 RGBW LED controller
+========================
+
+| The H801 is pretty afordable and easy to hack and adapt to your needs. It can be found on `Aliexpress <https://s.click.aliexpress.com/e/bbnUDBZW>`__

```

Comment:
Why the | in front?

Reply:
From reading some guidelines and making things stay on split up on lines like desired a guide said to use the | symbol in front, having some issues finding a good WYSIWG editor for RST and the docker image for esphomedocs didnt work for me :(